jenkins/scripts/integration_test.sh

```diff
@@ -37,11 +37,8 @@ CAPM3_VERSION="${CAPM3_VERSION:-v1beta1}"
 NUM_NODES="${NUM_NODES:-2}"
 TESTS_FOR="${TESTS_FOR:-integration_test}"
 BARE_METAL_LAB=false
-
 DEFAULT_HOSTS_MEMORY="${DEFAULT_HOSTS_MEMORY:-4096}"
-
-VM_TIMELABEL="${VM_TIMELABEL:-$(date '+%Y%m%d%H%M%S')}"
```

Comment:
Is there any specific reason we do not want the time tag to be appended to the executer VM name?

Reply:
yes, we were relayin on that in case we need to give a name to VM within this script. But in reality we never do it, because naming of the VM takes place earlier than we reach here, in the pipeline. And in the integration test pipeline we have a script that does the same, but in different env var `VM_KEY`

`VM_KEY = (1..4).collect { ('a'..'z').join("")[ rand.nextInt( 26 ) ] }.join("")`